ftw/news/lawgiver.zcml

```diff
@@ -0,0 +1,22 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:lawgiver="http://namespaces.zope.org/lawgiver"
+    i18n_domain="ftw.news">
+
+    <include package="ftw.lawgiver" file="meta.zcml" />
+
+    <lawgiver:map_permissions
+        action_group="add"
+        permissions="ftw.news: Add NewsFolder"
+        />
+
+    <lawgiver:map_permissions
+        action_group="add"
+        permissions="ftw.news: Add News"
```

Comment:
you could merge the two ``add`` groups into one statement, separating the permissions with `,\n `